spec/array_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'array'
+
+describe Array do
+  it {is_expected.to respond_to :molliesinject}
+
+  it 'can do simple addition' do
+    array = [1, 2, 3, 4, 5, 6, 7, 8]
+    expect(array.molliesinject{|tot, num| tot + num}).to eq(array.inject{|tot, num| tot + num})
```

Comment:
Line is too long. [95/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.